README.md

```diff
@@ -131,6 +131,8 @@ You can change the default aliases by defining these variables below.
 ``` bash
 forgit_log=glo
 forgit_diff=gd
+forgit_diff_cached=gdca
```

Comment:
I think I would like gdc to mirror gds

Reply:
I chose gdca to align with the git plugin in omz. I can change to gdc if preferred.